ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -209,6 +210,24 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		30531896244020B4005AE004 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)",
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n${PODS_ROOT}/Fabric/run\n\n";
```

Comment:
Purely for project config cleanliness.
The comment in the script doesn’t make sense in here as it’s run in the build phase. Can we please remove it? 
Doesn’t have to be done in this PR

Reply:
Yep ... it was the default. I'll remove 👍

Reply:
Removed comment.